### ISSUE TITLE
Twilio Support

### ISSUE THREAD

**Issue by askhogan** [blak3r/yaai#4]
Add twilio support [enhancement]

**Comment by blak3r** [comment]
@askhogan How's your twilio integration coming?  Just curious.

**Comment by askhogan** [comment]
Good.  I am not finished yet, but do plan to complete the changes.  My life is pretty frozen right now.  I just graduated from law school and take the CA bar exam at the end of July.  I am in a caveman mode until then.

I committed my changes to my fork here if you want to take a look https://github.com/askhogan/yaai

**Comment by blak3r** [comment]
@askhogan good luck!

**Comment by askhogan** [comment]
Back!  I am going to do some work on this... this weekend.  I am considering scrapping my User Interface - the one I showed you at SugarCon and just running with your User Interface so I don't have to work on my project internally and this project separately.  Should get some time at the end of the week.  

How is Baltimore?

**Comment by blak3r** [comment]
Cool!  Good to hear from ya.  How'd the bar go?
I thought that scraping your interface was always your plan.  You don't have to work on this project for my sake :)

I think a lot more enhancements are in the woodwork that as long as you take care of the logger will work out of the box.

Baltimore is good... volleyball season is going well.  Getting psyched for Ravens games again!

**Comment by askhogan** [comment]
It went well.  It was tough and a little surreal spending so much time on something that will never get published or be of any practical value to anyone.  I spent 2 months studying and 3 days in the exam room, however, the exam reviewer will most likely spend 16 minutes grading it and then dispose of it.  

It was my plan I just ran into a few issues on my end that I have since resolved.  

Cool.  Excited to look at it!  

Tall people shouldn't play sports.  It just makes shorter people look bad.  So rude.
